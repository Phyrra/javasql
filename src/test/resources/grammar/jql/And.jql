field1 = 1 and field2 = 2