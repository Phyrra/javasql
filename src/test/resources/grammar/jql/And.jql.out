([field1] = 1 AND [field2] = 2)